src/client/app/boot.js

```diff
@@ -99,8 +103,7 @@
 	// If mobile, insert the viewport meta tag
 	if (isMobile) {
 		const viewport = document.getElementsByName("viewport").item(0);
-		viewport.setAttribute('content',
-			`${viewport.getAttribute('content')},minimum-scale=1,maximum-scale=1,user-scalable=no`);
+		viewport.content = `${viewport.getAttribute('content')},minimum-scale=1,maximum-scale=1,user-scalable=no`;
```

Comment:
I think you can also use getter of content propery.

```suggestion
		viewport.content = `${viewport.content},minimum-scale=1,maximum-scale=1,user-scalable=no`;
```

Reply:
You're right! My bad!